### PR TITLE
feat: add suport for deserialization

### PR DESCRIPTION
**Pull request by cezarsmpio**
### Links
image

### Lists
image

### Tables
image

### CODE REVIEW

File: packages/rich-text/src/plugins/EmbeddedEntityInline/index.tsx [hunk 1]
```diff
@@ -61,7 +62,7 @@ function EmbeddedEntityInline(props: EmbeddedEntityInlineProps) {
   }
 
   return (
-    <span {...props.attributes} className={styles.root}>
+    <span {...props.attributes} className={styles.root} data-emebedded-entity-inline-id={entryId}>
```
Comment (cribbles) [inline packages/rich-text/src/plugins/EmbeddedEntityInline/index.tsx:65]: typo: should be `embedded` (also below)

File: packages/rich-text/src/plugins/Hyperlink/index.tsx [hunk 2]
```diff
@@ -63,6 +64,79 @@ export function createHyperlinkPlugin(sdk: FieldExtensionSDK): SlatePlugin {
...
     onKeyDown: buildHyperlinkEventHandler(sdk),
+    deserialize: (editor) => {
+      const hyperlinkOptions = getSlatePluginOptions(editor, INLINES.HYPERLINK);
+      const entryHyperlinkOptions = getSlatePluginOptions(editor, INLINES.ENTRY_HYPERLINK);
+      const assetHyperlinkOptions = getSlatePluginOptions(editor, INLINES.ASSET_HYPERLINK);
+
+      const isAnchor = (element) =>
+        element.nodeName === 'A' &&
+        !!element.getAttribute('href') &&
+        element.getAttribute('href') !== '#';
```
Comment (cribbles) [inline packages/rich-text/src/plugins/Hyperlink/index.tsx:75]: What's wrong with that? Is this something that comes up a lot in gdocs `href`s? (I get that the link wouldn't point anywhere -- even so...

I also wonder if we should think about js injection here via `javascript:`. But not something to capture in this PR and I don't think it's something we care about in the legacy editor anyhow.

**Review by cribbles — APPROVED** [review]
left a nit issue and a question, nothing blocking though, in general LGTM 👍

File: packages/rich-text/src/plugins/Hyperlink/index.tsx (hunk 2 above)
Comment (cezarsmpio) [reply packages/rich-text/src/plugins/Hyperlink/index.tsx:75]: Nothing is wrong to be honest... maybe we should validate if the `href` attribute is a valid URL?

**Pull request merged**